benchmark/solver/solver.cpp

```diff
@@ -167,6 +167,20 @@ const std::map<std::string, std::function<std::unique_ptr<gko::LinOpFactory>(
                             .with_preconditioner(give(precond))
                             .on(exec);
                     }},
+                   {"lower_trs",
+                    [](std::shared_ptr<const gko::Executor> exec,
+                       std::shared_ptr<const gko::LinOpFactory> precond) {
+                        return gko::solver::LowerTrs<>::build()
```

Comment:
Just a question, we dont use a template type here ? Maybe it is better to add `ValueType` to all template parameters ?

Reply:
That's a valid point! However, I believe we should address this in another PR